tools/sklearn/clf_metrics.xml

```diff
@@ -32,8 +32,13 @@ y_p = columns("$clf_metrics.infile2","$clf_metrics.col2")
 
 options = params["clf_metrics"]["options"]
```

Comment:
Can you manage to put these changes in a separate PR please, as well es the macro changes.